static_src/actions/service_actions.js

```diff
@@ -45,6 +45,37 @@ export default {
     });
   },
 
+  createInstanceDialog(serviceGuid, planGuid) {
```

Comment:
is the `createInstance_Dialog_` the best place for it in the rest of the service actions?